### ISSUE TITLE
Outdated content emails are being sent to blocked users

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#15367]
## Describe the defect
During an audit of the number of emails sent, I discovered that email notifications are being sent to blocked users.

The query that gets the users is supposed only get active users is asking for users with a status of 1. Like it should.
https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/docroot/modules/custom/va_gov_notifications/src/Service/OutdatedContent.php#L311C3-L317C6

For some reason `->condition('status', 1)` is resulting in both active and blocked users being retrieved.

We should only be sending slightly less than  370 VAMC notifications, but we will (on Oct 1) be sending ~550 VAMC notifications
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/5752113/9e1f6118-d6a6-487d-8db4-6c98bb22cb8b)

The reason it is not exactly 378 count from the [filtered people View](https://prod.cms.va.gov/admin/people?user=&status=1&role=All&workbench_access_section__section=8) is there are some people that appear in that list multiple times if they are assigned to multiple facilities, they will only receive 1 email.

This same defect will also send vet center notifications to vet center editors who are blocked.

The impact is small if we assume everyone that is blocked is no longer with the VA.  The email will just get eaten.  But if they have just changed offices, we are essentially spamming them.

This is not a new defect, this has been present from the beginning of our outdated content notifications.

## AC / Expected behavior
- [ ] an outdated content notification will only be sent to an active (unblocked) user who has outdated content.

## Screenshots
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/5752113/c5251717-84a5-42da-bedb-3776a3616b88)

 ## Additional context
Add any other context about the problem here. Reach out to the Product Managers to determine if it should be escalated as critical (prevents users from accomplishing their work with no known workaround and needs to be addressed within 2 business days).

## Labels
(You can delete this section once it's complete)
- [x] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [x] CMS workstream (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

**Comment by swirtSJW** [comment]
This query that should only be returning enabled users, is actually returning all users
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/5752113/986627e6-56be-48af-b367-96667a0afe6c)

This might actually be a core bug.

**Comment by swirtSJW** [comment]
Closing this as it is only due to it running locally. Disregard my ramblings about mailing blocked users. It only happens on local sandboxes because part of our database copy process enables ALL the users. So the 550 count was coming from my local. When run on prod it will be closer to the 370.
